src/main/java/hw_3/Task1.java

```diff
@@ -0,0 +1,14 @@
+package hw_3;
+
+import java.util.Arrays;
+
+public class Task1 {
+    public static void main(String[] args) {
+        double[] numbers = {3.43, 12.1212, 12.22, 2.29};
+
+        String result = Arrays.toString(numbers)
+                              .replace(", ", ";");
+
+        System.out.println(result);
```

Comment:
Realization is incorrect. You need to use cycle here to work with each element from numbers array. And combine each element with string and ; symbol

Reply:
Done